int32 frame
PersonPose[] poses